### PR TITLE
Add support for Debian Bookworm

### PR DESCRIPTION
**Pull request by javierm**
## Objectives

* Make it possible to install Consul Democracy in the latest stable version of Debian

### CODE REVIEW

**Comment by Senen** [comment]
Hi @javierm, I added a commit to fix the database permissions error when using PostgreSQL 15. Let me know what you think!

**Review by Senen — APPROVED** [review]

File: roles/postgresql/tasks/main.yml [hunk 1]
```diff
@@ -17,17 +17,26 @@
...
+        name: "{{ database_name }}"
+        owner: "{{ database_user }}"
+
+    - name: Grant {{ database_user }} user access to database {{ database_name }}
+      postgresql_privs:
+        type: database
+        database: "{{ database_name }}"
+        roles: "{{ database_user }}"
+        grant_option: no
+        privs: ALL
```
Comment (javierm) [inline roles/postgresql/tasks/main.yml:39]: @Senen I've run the tests without these lines, and they pass :thinking:. Are these lines necessary, or is it enough to change the order and create the user and then the database?

Comment (Senen) [reply roles/postgresql/tasks/main.yml:39]: Yes, I checked the `deploy` permissions for tables, schemas and with and without this task and there is no difference.

Comment (Senen) [reply roles/postgresql/tasks/main.yml]: I've just pushed a version removing the task.

**Pull request merged**